### ISSUE TITLE
Support for Apollo @defer Directive

### ISSUE THREAD

**Issue by karthikraobr** [99designs/gqlgen#381]
I know this is still in alpha, but are there plans to support the [following?](https://www.apollographql.com/docs/react/features/defer-support.html)

**Comment by mathewbyrne** [comment]
No plans, especially since it's not part of the core spec.  gqlgen does already run resolvers in parallel, which might help reduce the need for this in the first place.  Implementing this as a plugin would be neat, but quite difficult given our current plugin architecture thoughts.

Also here is a link the server implementation docs: https://github.com/apollographql/apollo-server/blob/defer-support/docs/source/defer-support.md

**Comment by fasibio** [comment]
https://graphql.org/blog/2020-12-08-improving-latency-with-defer-and-stream-directives/

Is it now a part of the core definition ?

**Comment by frederikhors** [comment]
@fasibio can you open a https://github.com/99designs/gqlgen/discussions/categories/ideas so we can track this?
